tests/vcd_vapp.py

```diff
@@ -116,5 +116,56 @@ def test_110_instantiate_vapp_identical(self):
         assert len(vm) > 0
         assert vm[0].get('name') == self.config['vcd']['vm']
 
+    def test_111_instantiate_vapp_custom_disk_size(self):
+        logged_in_org = self.client.get_org()
+        org = Org(self.client, resource=logged_in_org)
+        v = org.get_vdc(self.config['vcd']['vdc'])
+        vdc = VDC(self.client, href=v.get('href'))
+        assert self.config['vcd']['vdc'] == vdc.get_resource().get('name')
+        result = vdc.instantiate_vapp(self.config['vcd']['vapp'],
+                                      self.config['vcd']['catalog'],
+                                      self.config['vcd']['template'],
+                                      network=self.config['vcd']['network'],
+                                      fence_mode='bridged',
+                                      deploy=True,
+                                      power_on=False,
+                                      disk_size=self.config['vcd']['disk_size_new'])
```

Comment:
Add new config params(disk_size_new here) in config.yml. Others can run these tests after just changing the value.